<launch>
<node pkg="sas_robot_driver_denso" type="sas_robot_driver_denso_node" name="vs050_1">
<rosparam file="$(find robot_control)/cfg/sas_robot_driver_denso_vs050_1.yaml" command="load"/>
</node>
</launch>
